.travis.yml

```diff
@@ -0,0 +1,18 @@
+language: php
+
+php:
+  - 7.1
+
+cache:
+  directories:
+    - vendor
+
+install:
+  - travis_retry composer install --no-interaction --prefer-source
+
+script:
+  - vendor/bin/php-cs-fixer fix --dry-run -v
```

Comment:
Why not run these through composer?

Reply:
No specific reason just copied this from another more up to date public lib of LinioIT.
There are also some differences.

Reply:
I would use `composer lint:check` here. This way the same thing is used in CI as is used locally. It allows us to control it from a single place.